### ISSUE TITLE
[BUG]: Image/s are deleted from the component after translate from default language

### ISSUE THREAD

**Issue by Shekhar-Zealous** [Fekide/strapi-plugin-translate#232]
**Describe the bug**
Image is deleted from inside of the populated component original record right after it is translated from default locale into Spanish language.

A new record is created in a collection having default language as English. Right after the record is saved with populated components having images. Right after the record is saved with populated components having images

**To Reproduce**
1. Create a default English record for pages (https://prnt.sc/mKVmCENY_JmO) and add data with populated component having image/s (https://prnt.sc/pMqVlaQWf5CJ).
2. Click on "Locale" dropdown from Action list and move to Spanish language (https://prnt.sc/uLQsxmvrqdtS). Then click on "Translate from another locale" from English and click "Yes, fill in" and save the record. 
3. Scroll down to the SEO component and you will find image is copied in Spanish record (https://prnt.sc/O-54oV-LgqQx) but the image from the English record - SEO component  will be removed (https://prnt.sc/mdGtyZVwNb8S)

**Expected behavior**
The image should be copied and not deleted from the translated record.

**System (please complete the following information):**
 - OS: Win 10
 - Browser: Chrome
 - Versions:
   - strapi: 4.9.0
   - strapi-plugin-translate: 1.0.4
   - provider: strapi-provider-translate-deepl@1.0.4

**Plugin config**
Please add your plugin config here, e.g.
```js
module.exports = {
  translate: {
    enabled: true,
    config: {
      provider: 'deepl',
      providerOptions: {
        apiKey: '*************'
      },
      translatedFieldTypes: [
        'string',
        { type: 'text', format: 'plain' },
        { type: 'richtext', format: 'markdown' },
        'component',
        'dynamiczone',
      ],
      translateRelations: true,
    },
  },
}
```

**Comment by sargreal** [comment]
@Shekhar-Zealous I don't seem to be able to reproduce this, can you please verify if this still happens with the latest version of the plugin?

**Comment by Shekhar-Zealous** [comment]
> @Shekhar-Zealous I don't seem to be able to reproduce this, can you please verify if this still happens with the latest version of the plugin?

Yes, I have just checked it by upgrading the package to latest version 1.1.6 and it is still happening.
